src/models.py

```
@@ -376,26 +383,36 @@ class ActiveQueueSession(Base):
+
     last_updated: Mapped[datetime.datetime] = mapped_column(
         DateTime(timezone=True), default=func.now(), onupdate=func.now()
     )
 
+
+@final
+class WebhookEndpoint(Base):
+    __tablename__: str = "webhook_endpoints"
+    id: Mapped[str] = mapped_column(TEXT, default=generate_nanoid, primary_key=True)
+    workspace_name: Mapped[str] = mapped_column(
+        ForeignKey("workspaces.name"), index=True, nullable=False
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)